.k8s/kibana.yaml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
I know both are fine, but `yml` for the file extension is generally what Elasticsearch uses.

Reply:
This is not a `Elasticsearch` resource, and personally i don't think `yml` is fine, but a lazy abbreviation of well-defined extension:
```
1. Is there an official extension for YAML files?

Please use ".yaml" when possible.
```
font: https://yaml.org/faq.html

About `Elasticsearch` resource files there's nothing i can do, but for others i can at least advocate.